containers/src/Data/Sequence/Internal.hs

```diff
@@ -1888,7 +1888,7 @@ snocTree' (Deep s pr m (One a)) b =
 (><)            :: Seq a -> Seq a -> Seq a
 Seq xs >< Seq ys = Seq (appendTree0 xs ys)
 
--- The appendTree/addDigits gunk below is machine generated
+-- The appendTree/addDigits gunk below is machine generated via mkappend.hs
```

Comment:
Please point out that while it was originally generated thus, it has since been edited to add strictness annotations.

Reply:
good call